docs-src/devguide.md

```diff
@@ -37,11 +40,13 @@ for this project:
 * [Open issues][gh-issues]
 * [Project dashboard][gh-dashboard]
 
-[gh-issues]: https://github.com/kubernetes-sigs/service-apis/issues
-[gh-dashboard]: https://github.com/kubernetes-sigs/service-apis/projects/1
-
 Issues labeled `good first issue` and `help wanted` are especially good for a
 first contribution.
+Please take a look at [milestones][gh-milestones] as well.
```

Comment:
Not quite sure how to say this, but I think you could provide a bit more detail here.

```suggestion
We use [milestones][gh-milestones] to track our progress towards releases. Looking at our current milestone can help identify our highest priority issues.
```

Should we also link to releases here?